client/my-sites/themes/theme-download-card/index.jsx

```diff
@@ -0,0 +1,40 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import i18n from 'lib/mixins/i18n';
+
+var ThemeDownloadCard = React.createClass( {
+
+	propTypes: {
+		theme: React.PropTypes.string.isRequired,
+		href: React.PropTypes.string
+	},
+
+	render() {
+		var downloadURI = this.props.href || ( 'https://downloads.wordpress.org/theme/' + this.props.theme + '.zip' );
+		var downloadText = i18n.translate( 'This theme is available for download to be used on your {{a}}WordPress self-hosted{{/a}} installation.', {
+			components: {
+				a: <a href={ downloadURI } />
```

Comment:
I think maybe the should be something like https://wordpress.org/ rather than the download link (see https://github.com/Automattic/wp-calypso/pull/4473#discussion_r58350369)